examples/stable_diffusion_xl/scripts/run_infer_base_controlnet.sh

```diff
@@ -0,0 +1,12 @@
+export MS_PYNATIVE_GE=1
```

Comment:
can remove it since it's hard-coded lately

Reply:
removed